tests/runner.py

```diff
@@ -1031,16 +1043,14 @@ def do_run(self, src, expected_output, args=[], output_nicerizer=None,
       test_index += 1
 
   def get_freetype_library(self):
-    self.set_setting('DEAD_FUNCTIONS', self.get_setting('DEAD_FUNCTIONS') + ['_inflateEnd', '_inflate', '_inflateReset', '_inflateInit2_'])
```

Comment:
What happened to this setting?

Reply:
Oh, this was some old hack I think - we haven't needed it for a long time.